test/corecel/math/RegulaFalsiRootFinder.test.cc

```diff
@@ -0,0 +1,86 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020-2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file corecel/math/RegulaRootFinder.test.cc
+//---------------------------------------------------------------------------//
+#include "corecel/math/RegulaFalsiRootFinder.hh"
+
+#include <cmath>
+#include <functional>
+
+#include "corecel/Constants.hh"
+#include "corecel/Types.hh"
+#include "orange/OrangeTypes.hh"
+
+#include "celeritas_test.hh"
+
+namespace celeritas
+{
+using constants::pi;
+
+namespace test
+{
+
+// Solve: (x-2)(x+2) = 0
+TEST(RegulaFalsi, root_two)
+{
+    auto root = [](real_type t) { return (t - 2) * (t + 2); };
+
+    RegulaFalsi find_root{root, 1e-13};
```

Comment:
These tolerances are too tight for a single-precision solve... perhaps include the `corecel/math/SoftEqual.hh` header and use `constexpr real_type tol = SoftEqual<>{}.rel();`?